kvmagent/kvmagent/plugins/tensorfusion/monitor.py

```
@@ -0,0 +1,277 @@
+
+            try:
+                new_worker = self._executor.start(req)
+                new_worker.restarting = False
+                replaced = self._store.replace(new_worker, expected_worker=current_worker)
+                if replaced is None:
+                    self.clear(device_uuid)
+                    logger.warn('WorkerRestartMonitor: worker %s changed during restart, stopping orphan replacement' %
+                                device_uuid)
+                    try:
+                        self._executor.stop(new_worker)
+                    except Exception as stop_error:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)